renormalizer/mps/mp.py

```diff
@@ -934,6 +934,38 @@ def _array2mt(self, array, idx, allow_dump=True):
     
     def build_empty_mp(self, num):
         self._mp = [[None]] * num
+    
+    def dump(self, fname, other_attrs=[]):
```

Comment:
Not a big fan of using `[]` as the default argument (see [this post](https://www.toptal.com/python/top-10-mistakes-that-python-programmers-make))

Reply:
fixed